.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python: 3.7
+sudo: true
```

Comment:
Travis has ignored `sudo` since the last ~12 months, and it is implicitly `true` even if you set it to `false`.

Proof: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

As you see, they abandoned the container-based environments so that both `sudo: false` and `sudo: true` builds run with the same speed.

That being said, the current Travis config is not using sudo anyway, so I can turn it off, but that will lead to `pip install --user` in the `install` section. I don't think it's worth it.